glide.yaml

```diff
@@ -25,12 +25,12 @@ import:
   version: ~0.6.2
   subpackages:
   - data
-- package: github.com/tendermint/merkleeyes
-  version: master
+- package: github.com/tendermint/iavl
+  version: develop
   subpackages:
   - iavl
 - package: github.com/tendermint/tmlibs
-  version: ~0.3.2
+  version: develop
```

Comment:
won't these changes make `go get` easily breakable? also contradicts: https://github.com/tendermint/coding/issues/24

Reply:
only on develop, which is fine. we will change these back to tags when we'll be doing a release

Reply:
ah gotcha

Reply:
exactly. we can include that in the guidelines